code/mob/living/carbon/human/procs/Life.dm

```diff
@@ -934,19 +934,18 @@
 				for (var/datum/gas/rad_particles/RV in breath.trace_gases)
 					src.changeStatus("radiation", RV.moles, 2 SECONDS)
 
-		if (breath.temperature > (T0C+66) && !src.is_heat_resistant()) // Hot air hurts :(
-			if (!has_cyberlungs || (has_cyberlungs && (breath.temperature > (T0C+500))))
-				var/burn_damage = min((breath.temperature - (T0C+66)) / 3,10) + 6
-				TakeDamage("chest", 0, burn_damage, 0, DAMAGE_BURN)
-				if (prob(20))
-					boutput(src, "<span class='alert'>You feel a searing heat in your lungs!</span>")
-					if (src.organHolder)
-						src.organHolder.damage_organs(0, max(burn_damage, 3), 0, list("left_lung", "right_lung"), 80)
-
-				hud.update_fire_indicator(1)
-				if (prob(4))
-					boutput(src, "<span class='alert'>Your lungs hurt like hell! This can't be good!</span>")
-					//src.contract_disease(new/datum/ailment/disability/cough, 1, 0) // cogwerks ailment project - lung damage from fire
+		if (breath.temperature > min(organHolder.left_lung.temp_tolerance, organholder.left_lung.temp_tolerance) && !src.is_heat_resistant()) // Hot air hurts :(
+			var/burn_damage = min((breath.temperature - (T0C+66)) / 3,10) + 6
```

Comment:
Should this not use the (averaged?) temp_tolerance of lungs?

Reply:
this is just a threshold, meaning min.
no matter how much damage it deals, the weakest lung should not have immunity due to the strongest lung.

Reply:
Which is to say, it is not using the tolerance at all, just T0C+66

Reply:
damage isn't using tolerance at all, because that'd be a feature out of scope for this PR.
threshold acts identically to before except in the case of having only one lung, which happens to be a cyberlung. that is mostly in scope.